src/observations/process_radar.jl

```diff
@@ -30,7 +28,7 @@ See https://doi.org/10.1103/PhysRevLett.13.789.
 """
 function shapiro_delay(e, p, q)
     shap = 0.0 # 2μ[1]/(c_au_per_day^2)
-    shap_del_days = (2PlanetaryEphemeris.μ[su]/(c_au_per_day^3))*log( (e+p+q+shap)/(e+p-q+shap) ) # days
+    shap_del_days = (2PE.μ[su]/(c_au_per_day^3))*log( (e+p+q+shap)/(e+p-q+shap) ) # days
```

Comment:
For clarity, I think it is better to use here `2μ_DE430[su]`, since that *alias* is already a constant. This comment applies also below.

Reply:
Done.